sdk/clientcore/core/src/main/java/module-info.java

```diff
@@ -20,6 +20,8 @@
     exports io.clientcore.core.util.binarydata;
     exports io.clientcore.core.util.configuration;
     exports io.clientcore.core.util.serializer;
+    exports io.clientcore.core.util.auth;
+    exports io.clientcore.core.implementation.util.auth;
```

Comment:
Need to verify if we should move concrete implementations of ChallengeHandler to Okhttp Lib or better to export ?